scripts/release/release.go

```diff
@@ -1,153 +0,0 @@
-package main
```

Comment:
If this is being removed anyway, I'm curious why we didn't write the patch-release code over this to preserve git history

Reply:
/hold

still curious about this

Reply:
Hi @haircommander , apologies for the late reply.
We decided to focus on a patch releases for now, just to see how the workflow will work and we agreed that the file is not in use and so we should remove it.

Reply:
@saschagrunert @kwilczynski are you opposed to making the changes in place instead of creating a new file to preserve git history?

Reply:
Hm, no we can replace the file if you want.

Reply:
We should keep the old file for now. We can remove it once things are ready. This gives a clear commit then.

Reply:
reverted the file as requested. @haircommander